mkosi/__init__.py

```diff
@@ -4848,6 +4848,21 @@ def parse_remove_files(value: str) -> List[str]:
     return ["/" + os.path.normpath(p).lstrip("/") for p in value.split(",") if p]
 
 
+def parse_ssh_agent(value: str) -> Union[Path, bool]:
+    """Can be True/False or a path to a socket."""
```

Comment:
This isn't quite correct, no? The return value of this function will be either `False` or the path `SSH_AUTH_SOCK` points, too?